extension/src/setup/studio.test.ts

```diff
@@ -0,0 +1,97 @@
+import * as Fetch from 'node-fetch'
+import { fetchStudioToken } from './studio'
+import { Toast } from '../vscode/toast'
+
+const mockedFetch = jest.mocked(Fetch)
+const mockedFetchDefault = jest.fn()
```

Comment:
Hmmmm, this code gives me an error: `mockedFetch.mockImplementation is not a function`. Maybe it's because because fetch is a function, not a source object?